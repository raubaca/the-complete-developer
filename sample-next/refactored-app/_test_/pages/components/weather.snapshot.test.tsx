/**
 * @jest-environment node
 */

import { act, create } from 'react-test-renderer';
import PageComponentWeather from '../../../pages/components/weather';

describe('PageComponentWeather', () => {
  test('renders correctly', async () => {
    let component: any;
    await act(async () => {
      component = await create(<PageComponentWeather></PageComponentWeather>);
    });
    expect(component.toJSON()).toMatchSnapshot();
  });
});
