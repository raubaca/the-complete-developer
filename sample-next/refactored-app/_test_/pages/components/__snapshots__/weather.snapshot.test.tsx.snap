// Jest Snapshot v1, https://jestjs.io/docs/snapshot-testing

exports[`PageComponentWeather renders correctly 1`] = `
<h1
  onClick={[Function]}
>
  The weather is 
  sunny
  , and the counter shows 
  1
</h1>
`;
